pyproject.toml

```diff
@@ -1,52 +1,227 @@
+    { name="Helio Chissini de Castro", email="helio.chissini.de.castro@cariad.technology" },
+]
+maintainers = [
+    { name="nexB. Inc. and others", email="info@aboutcode.org"},
+    { name ="Helio Chissini de Castro", email="helio.chissini.de.castro@cariad.technology" },
+]
+readme = "README.md"
+license = "Apache-2.0"
+license-files = [
+    "apache-2.0.LICENSE",
+    "NOTICE",
+    "AUTHORS.rst",
+    "CHANGELOG.rst",
+    "CODE_OF_CONDUCT.rst",
+]
+requires-python = ">= 3.10"
+keywords = [
+    "dependencies",
+    "dependency resolution",
+    "dependency",
+    "open source",
+    "package",
+    "pip",
+    "pypi",
+    "python",
+    "requirements",
+    "resolvelib",
+    "resolver",
+    "SBOM",
+    "sca",
+    "scan",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "aiofiles>=24.1.0",
+    "aiohttp>=3.11.14",
+    "click>=8.1.8",
+    "colorama>=0.4.6",
+    "commoncode>=32.2.1",
+    "dparse2>=0.7.0",
+    "importlib-metadata>=8.6.1",
+    "license-expression>=30.4.1",
+    "mock>=5.2.0",
+    "packageurl-python>=0.16.0",
+    "packaging>=24.2",
+    "pip-requirements-parser>=32.0.1",
+    "pkginfo2>=30.0.0",
+    "pydantic>=2.10.6",
+    "pydantic-settings>=2.8.1",
+    "resolvelib>=1.1.0",
+    "rich>=13.9.4",
+    "setuptools>=77.0.1",
+    "tenacity>=9.0.0",
+    "toml>=0.10.2",
+    "typecode>=30.0.2",
+]
+
+[project.urls]
+"Homepage" = "https://github.com/aboutcode-org/python-inspector"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/python_inspector", "src/_packagedcode"]
+
+[dependency-groups]
+dev = [
+    "hatch>=1.14.0",
+    "mypy>=1.15.0",
+    "pre-commit>=4.2.0",
+    "pycodestyle>=2.12.1",
+    "ruff>=0.11.0",
+    "types-requests>=2.32.0.20250306",
+    "types-toml>=0.10.8.20240310",
+]
+docs = [
+    "doc8>=1.1.2",
+    "sphinx>=7.4.7",
+    "sphinx-reredirects>=0.1.6",
+    "sphinx-rtd-theme>=3.0.2",
+]
+test = [
+    "aboutcode-toolkit>=11.0.0",
+    "pip>=25.0.1",
+    "pytest>=8.3.5",
+    "pytest-cov>=6.0.0",
+    "pytest-rerunfailures>=15.0",
+    "pytest-xdist>=3.6.1",
+    "twine>=6.1.0",
+]
+
+[tool.hatch.envs.default.scripts]
+validate-docs = "sh -c 'doc8 --max-line-length 100 --ignore-path docs/_build/ docs/'"
+
+[project.scripts]
+python-inspector = "python_inspector.resolve_cli:resolve_dependencies"
 
-[tool.setuptools_scm]
-# this is used populated when creating a git archive
-# and when there is .git dir and/or there is no git installed
-fallback_version = "9999.9999.99"
 
 [tool.pytest.ini_options]
-norecursedirs = [
```

Comment:
These are needed. Why remove these?

Reply:
Instead of doing huge multiple exclusions, used only pythonpath for the two required directories